### ISSUE TITLE
Validation set: test set of 300W-LP

### ISSUE THREAD

**Issue by yeqi17** [lippman1125/pytorch_FAN#7]
I have downloaded the 300W-LP dataset, but I can not find the test set in it. Could you tell me which dataset is the validation set in your training task.

**Comment by Jiyongma06** [comment]
Can you tell me where I can download 300W-LP dateset ? thanks
